src/style.css

```diff
@@ -1,48 +1,165 @@
+html,
 body {
-  background-color: rgb(78, 195, 199);
+  width: 100%;
+  height: 100%;
+  padding: 0%;
+  margin: 0%;
 }
 
-.toDoHolder {
+body {
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
-  background: white;
-  flex-direction: column;
-  margin-top: 210px;
-  width: 600px;
-  margin-left: 420px;
 }
 
-.header {
+.list-title {
+  width: 100%;
   height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-family: monospace;
+  font-size: 18px;
+  font-weight: 600;
 }
```

Comment:
- [ ] Kindly make sure to fix the delete function. When I select items and click on clear all selected it is not deleted the first time. You can see the video below for the error.

https://user-images.githubusercontent.com/70269507/166867148-9eedad94-165b-4d38-9dcf-6025c610f2ab.mov

- [ ] Kindly also fix the error which occurs when I tried running your app locally. Below is a screenshot of the terminal.
<img width="806" alt="Screenshot 2022-05-05 at 8 20 04 AM" src="https://user-images.githubusercontent.com/70269507/166867368-9a7185b7-0a8b-4d30-8982-f17b59b6ffa5.png">